lib/adequate_crypto_address/cryptos/ada.rb

```diff
@@ -0,0 +1,32 @@
+module AdequateCryptoAddress
+  class Ada
```

Comment:
All these files seem to be duplicated for each coin. It would be better to make a base class to inherit from so we can reuse methods.